modules/machineset-yaml-gcp.adoc

```diff
@@ -0,0 +1,85 @@
+/ Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating-machineset-gcp.adoc
+
+[id="machineset-yaml-gcp_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on GCP
+
+This sample YAML defines a MachineSet that runs in Google Cloud Platform (GCP_
+and creates nodes that are labeled with `node-role.kubernetes.io/<role>: ""`
+
+In this sample, `<infrastructureID>` is the infrastructure ID label that is
+based on the cluster ID that you set when you provisioned
+the cluster, and `<role>` is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+  name: <infrastructureID>-w-a <2>
```

Comment:
name: <infrastructureID>-w-a <1>

Reply:
@sunzhaohua2, you know what "w-a" means here?

Reply:
I think `w` means worker, `a` means zone `us-central1-a`

Reply:
Do you know how this role abbreviation is obtained? (@ingvagabund, do you know?)